common/app/http/Filters.scala

```diff
@@ -8,12 +8,21 @@ import common.ExecutionContexts
 import implicits.Responses._
 import model.ApplicationContext
 import play.api.http.HttpFilters
-import play.api.mvc.{EssentialFilter, Filter, RequestHeader, Result}
-import play.filters.gzip.GzipFilter
+import play.api.mvc._
+import play.filters.gzip.{GzipFilterConfig, GzipFilter}
 
 import scala.concurrent.Future
 
-class Gzipper(implicit val mat: Materializer) extends GzipFilter(shouldGzip = (_, result) => !result.header.isImage)
+class GzipperConfig() extends GzipFilterConfig {
+  val excludeFromGzip = List(
```

Comment:
Could we add a note here to say that this is to turn off gzipping in the response?